src/allmydata/test/web/test_webish.py

```diff
@@ -105,8 +115,8 @@ def test_form_fields(self):
                     in fs.keys()
                 },
                 Equals({
-                    b"foo": b"bar",
-                    b"baz": b"some file contents",
+                    "foo": "bar",
+                    "baz": b"some file contents",
```

Comment:
I could try hard to get `FieldStorage` to emit consistent types, if desired.

Reply:
:scream_cat: I guess that current split makes sense, so seems fine as is.